unlock-app/src/components/content/ConnectingContent.tsx

```diff
@@ -0,0 +1,23 @@
+import ConnectingWaas from '../interface/connect/ConnectingWaas'
+import { AppLayout } from '../interface/layouts/AppLayout'
+
+export const ConnectingContent = () => {
+  return (
+    <div>
+      <AppLayout
+        title="Connecting"
+        authRequired={false}
+        showLinks={false}
+        showConnectButton={false}
+      >
+        <span className="w-full max-w-lg text-base text-gray-700">
+          We are connecting to the Unlock Protocol, please be patient and do not
```

Comment:
What happens if the user refreshes?

Reply:
Also this UI is not great. Since we know the user is logging in, can we just load the "connecting" modal and have this content in there?